lib/plugins/create/templates/azure-nodejs/serverless.yml

```diff
@@ -17,12 +17,55 @@ service: azure-nodejs # NOTE: update this with your service name
 # Check out our docs for more details
 # frameworkVersion: "=X.X.X"
 
+# if you are using webpack for packaging
+# custom:
+#   webpack:
+#     keepOutputDirectory: true
```

Comment:
Just a reminder: If we decide to remove webpack lingo from the whole template we might want to remove this as well. Otherwise if we add a comment about webpack below we might want to keep it here.